tests/vmi_multiqueue_test.go

```diff
@@ -152,3 +180,13 @@ var _ = Describe("[sig-compute]MultiQueue", func() {
 		})
 	})
 })
+
+func newNetworkAttachmentDefinition(networkName string) *nadv1.NetworkAttachmentDefinition {
```

Comment:
This is nice, consider having a following PR to extract it to `libnet` and use it by others tests as well.

Reply:
Totally (was written on PR desc)
Follow PR sounds good